packages/lisk-p2p/test/unit/utils.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { expect } from 'chai';
+import { onlyDigits, validateIp, validatePort } from '../../src/utils';
+
+describe('utils', () => {
+	describe('#validateIp', () => {
+		const correctIp = '127.0.0.1';
+		const IpWithCharacters = '127k.0.0.1';
+		const IpWithLessBlocks = '127.0.0';
+		const IpWithSpaces = '127.0.0. 1';
```

Comment:
We don't need so many simple cases (one or two is enough?).

A more useful case would be related to IPv4 vs IPv6 formats.

Another case would be; what if the ip provided is not a string but an object or null.